File: system-test/dotnet/SystemTests/Latest/SmokeTests/External/ClockSmokeTest.cs
using SystemTests.Latest.Base;
using Xunit;

namespace SystemTests.Latest.SmokeTests.External;

public class ClockSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToClock()
    {
        await Scenario().Assume().Clock().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/latest/smoke/external/ClockSmokeTest.java
package com.mycompany.myshop.systemtest.latest.smoke.external;

import com.mycompany.myshop.systemtest.latest.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class ClockSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToClock() {
        scenario.assume().clock().shouldBeRunning();
    }
}


